src/lib/flight_tasks/tasks/Transition/FlightTaskTransition.cpp

```diff
@@ -60,6 +60,7 @@ void FlightTaskTransition::checkSetpoints(vehicle_local_position_setpoint_s &set
 
 bool FlightTaskTransition::update()
 {
+	FlightTask::update();
```

Comment:
`FlightTaskOffboard` and `FlightTaskOrbit` need to call the base update as well.
Also to make it fully correct you need to logically and with the return value like for example here: https://github.com/PX4/Firmware/blob/2fca03e1bda313f86fd96d05130e830b9a2ee220/src/lib/flight_tasks/tasks/Manual/FlightTaskManual.cpp#L47

Reply:
Thanks, I added it for `FlightTaskOffboard`, but orbit inherits from Altitude, so it's not required